libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/internal/NotificationTurnIconResources.kt

```diff
@@ -0,0 +1,1499 @@
+        private var turnIconOnRampSharpLeft: Int =
+            R.drawable.mapbox_ic_notification_turn_sharp_left
+        private var turnIconOnRampSharpRight: Int =
+            R.drawable.mapbox_ic_notification_turn_sharp_right
+        private var turnIconRamp: Int = R.drawable.mapbox_ic_notification_ramp
+        private var turnIconRotary: Int = R.drawable.mapbox_ic_notification_rotary
+        private var turnIconRotaryLeft: Int = R.drawable.mapbox_ic_notification_rotary_left
+        private var turnIconRotaryRight: Int = R.drawable.mapbox_ic_notification_rotary_right
+        private var turnIconRotaryStraight: Int = R.drawable.mapbox_ic_notification_rotary_straight
+        private var turnIconRotarySlightLeft: Int =
+            R.drawable.mapbox_ic_notification_rotary_slight_left
+        private var turnIconRotarySlightRight: Int =
+            R.drawable.mapbox_ic_notification_rotary_slight_right
+        private var turnIconRotarySharpLeft: Int =
+            R.drawable.mapbox_ic_notification_rotary_sharp_left
+        private var turnIconRotarySharpRight: Int =
+            R.drawable.mapbox_ic_notification_rotary_sharp_right
+        private var turnIconRoundabout: Int = R.drawable.mapbox_ic_notification_roundabout
+        private var turnIconRoundaboutLeft: Int = R.drawable.mapbox_ic_notification_roundabout_left
+        private var turnIconRoundaboutRight: Int =
+            R.drawable.mapbox_ic_notification_roundabout_right
+        private var turnIconRoundaboutStraight: Int =
+            R.drawable.mapbox_ic_notification_roundabout_straight
+        private var turnIconRoundaboutSlightLeft: Int =
+            R.drawable.mapbox_ic_notification_roundabout_slight_left
+        private var turnIconRoundaboutSlightRight: Int =
+            R.drawable.mapbox_ic_notification_roundabout_slight_right
+        private var turnIconRoundaboutSharpLeft: Int =
+            R.drawable.mapbox_ic_notification_roundabout_sharp_left
+        private var turnIconRoundaboutSharpRight: Int =
+            R.drawable.mapbox_ic_notification_roundabout_sharp_right
+        private var turnIconTurnLeft: Int = R.drawable.mapbox_ic_notification_turn_left
+        private var turnIconTurnRight: Int = R.drawable.mapbox_ic_notification_turn_right
+        private var turnIconTurnStraight: Int = R.drawable.mapbox_ic_notification_turn_straight
+        private var turnIconTurnSlightLeft: Int = R.drawable.mapbox_ic_notification_turn_slight_left
+        private var turnIconTurnSlightRight: Int =
+            R.drawable.mapbox_ic_notification_turn_slight_right
+        private var turnIconTurnSharpLeft: Int = R.drawable.mapbox_ic_notification_turn_sharp_left
+        private var turnIconTurnSharpRight: Int = R.drawable.mapbox_ic_notification_turn_sharp_right
+        private var turnIconUturn: Int = R.drawable.mapbox_ic_notification_uturn
+
+        /**
+         * apply turn icon to the builder.
```

Comment:
```suggestion
         * apply arrive turn icon to the builder.
```

The same thing applies to 👇